packages/edge-gateway-link/src/index.js

```diff
@@ -37,6 +38,21 @@ function serverError (error, request, env) {
   return addCorsHeaders(request, errorHandler(error, env))
 }
 
+/**
+ * Proxy POST requests to the UCANTO Server defined in the environment.
+ *
+ * @param {Request} request
+ * @param {import('./env').Env} env
+ * @returns {Promise<Response>}
+ */
+async function proxyPostRequest(request, env) {
+  const originRequest = new Request(request)
+  const url = new URL(request.url)
+  const targetUrl = `${env.UCANTO_SERVER_URL}${url.pathname}`
```

Comment:
This'll mitigate against double slash if `UCANTO_SERVER_URL` is defined with a trailing slash...

```suggestion
  const targetUrl = new URL(url.pathname, env.UCANTO_SERVER_URL)
```